src/components/workspace/index.tsx

```diff
@@ -2,6 +2,9 @@ import Resizable, { ResizableProps, ResizeCallback } from 're-resizable';
 import * as React from 'react';
 import { Prompt } from 'react-router';
 
+import SourcecastControlbar, {
+  ISourcecastControlbarProps
+} from '../sourcecast/SourcecastControlbar';
```

Comment:
Same as above. Let's try to use composition.

Reply:
Fixed by 9829b2e